tools/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd ../cypher-codemirror
```

Comment:
This line expects us to be in tools folder in order to run script. I would prefer that we run script from root project folder